pom.xml

```diff
@@ -3,8 +3,8 @@
   <modelVersion>4.0.0</modelVersion>
   <parent>
     <groupId>org.nuxeo</groupId>
-    <artifactId>nuxeo-ecm</artifactId>
-    <version>11.2-SNAPSHOT</version>
+    <artifactId>nuxeo-parent</artifactId>
+    <version>11.3-SNAPSHOT</version>
```

Comment:
SNAPSHOT is needed because we want to use https://github.com/nuxeo/nuxeo/pull/4300.

Reply:
The commits are present in Nuxeo `11.3.35`.